railties/lib/rails/generators/rails/app/templates/config/initializers/request_forgery_protection.rb

```diff
@@ -0,0 +1,4 @@
+# Be sure to restart your server when you modify this file.
+
+# Enable origin-checking CSRF mitigation.
+Rails.application.config.forgery_protection_origin_check = true
```

Comment:
This need to be `Rails.application.config.action_controller.forgery_protection_origin_check = true`

This config needs to be documented in the configuring guide too.